frontend/src/scenes/surveys/SurveyAppearance.scss

```diff
@@ -121,7 +121,15 @@
 .description {
     margin-top: 5px;
     font-size: 13px;
-    opacity: 0.6;
+    color: rgba(var(--textColor), 0.6);
```

Comment:
This is a trick we use elsewhere in the application already. `--textColor` is expected to contain a string with the rgb-components of the color which then allows us to set an opacity, resulting in a color like `rgba(0, 0, 0, 0.6)`.